.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,38 @@
+name: CIFuzz
+on:
+  pull_request:
+  workflow_dispatch:
+- pull_request
```

Comment:
looks like this is causing an invalid file error on yaml.   I think the -pull_request shouldn't be there